src/alfrescoApi.ts

```diff
@@ -537,6 +537,10 @@ export class AlfrescoApi implements EventEmitter.Emitter {
         return this.config.authType === 'OAUTH';
     }
 
+    isPublicUrl(): boolean {
```

Comment:
Is better to call before (this.isOauthConfiguration())

Reply:
Thx :)

Reply:
are you sure you need to check it? this.oauth2Auth &&

Reply:
It looks like oauth2Auth could be undefined if no parameter was provided in the constructor 
https://github.com/Alfresco/alfresco-js-api/blob/development/src/alfrescoApi.ts#L52